src/client/features/enrichment/index.js

```diff
@@ -124,7 +128,7 @@ class Enrichment extends React.Component {
     let network = h('div.network', {
         className: classNames({
           'network-loading': loading,
-          'network-sidebar-open': true
+          'network-sidebar-open': activeMenu !== 'closeMenu'
```

Comment:
I think this is the only change that is needed.  Can you confirm if it works with only this js change?